prow/metrics/http.go

```diff
@@ -27,6 +27,7 @@ import (
 	"k8s.io/test-infra/prow/simplifypath"
 )
 
+// HttpRequestDuration returns a histogramVector with relevant fields set
```

Comment:
```suggestion
// HttpRequestDuration returns a histogram vector with relevant fields set
```

Like the other one :)